benchmark/opperf/README.md

```diff
@@ -173,6 +173,22 @@ This utility queries MXNet operator registry to fetch all operators registered w
 However, fully automated tests are enabled only for simpler operators such as - broadcast operators, element_wise operators etc... For the purpose of readability and giving more control to the users, complex operators such as convolution (2D, 3D), Pooling, Recurrent are not fully automated but expressed as default rules.
 See `utils/op_registry_utils.py` for more details.
 
+## Use nativa python timer
```

Comment:
nitpick: native (also acc to our naming convention we have 2 parameters
`native` and `python` where (`native` resembles MXNet's CPP pased profiler) and `python` indicates the python's time function